### PR TITLE
Adds connect API documentation

### PR DESCRIPTION
**Pull request by joepour**

### CODE REVIEW

File: connect/v1.0/connect.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# ClickView Connect API
+
+# RSS Feed
```
Comment (MrSmoke) [inline connect/v1.0/connect.md]: says RSS Feed

File: connect/v1.0/connect.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+The ClickView Connect API is used to securly obtain the information about the signed in user or school. Please contact ClickView to obtain a `clientId` if you wish to use this API.
```
Comment (MrSmoke) [inline connect/v1.0/connect.md]: `securely`

File: connect/connect.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# ClickView Connect API
```
Comment (MrSmoke) [inline connect/connect.md]: The link to this file is missing from the main readme

Comment (joepour) [reply connect/connect.md]: Added

File: connect/v1.0/connect.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+GET https://online.clickview.com.au/v1/connect
+GET https://online.clickview.co.uk/v1/connect
+GET https://online.clickview.co.nz/v1/connect
+```
+
+## Parameters
+
+| Name | Required | Description | Default | Example |
+| ---- | -------- | ----------- | ------- | ------- |
+| clientId | required | You clientId || `F437FEA9-F21A-4CA6-9C2F-E1792CF87CFE` |
```
Comment (MrSmoke) [inline connect/v1.0/connect.md]: Typo in you (Your `clientId`)

**Pull request merged**